### ISSUE TITLE
GATK doesn't work on Java 1.9

### ISSUE THREAD

**Issue by tseemann** [ngs-fzb/MTBseq_source#18]
I was getting this error:

```
##### ERROR MESSAGE: Invalid command line: Malformed walker argument: Could not find walker with name: RealignerTargetCreator
```

Tracked it down to here:

https://gatkforums.broadinstitute.org/gatk/discussion/10464/invalid-command-line-malformed-walker-argument

it seems it needs older Java 1.8 :-(

PS. picard seems to be ok

**Comment by cutpatel** [comment]
Yes, gatk unfortunately requires JAVA 8 and no other version is allowed.

**Comment by tseemann** [comment]
Are you able to add a `--java` option or honour a `MTBSEQ_JAVA` environment variable?
As Java 9 is now standard.

**Comment by elbakrifz** [comment]
I get the same error when I running : java -jar /Users/mac/Downloads/package-archive-gatk-GenomeAnalysisTK-3.7-0-gcfedb67/GenomeAnalysisTK.jar  -T RealignerTargetCreator  -R /Users/mac/Desktop/LmjFwholegenome_20070731_V5.2.fasta   -I /Users/mac/Desktop/NGS/marked-duplicates42.bam   -O SRR6369642_realtarget.list with the java 8 update 251, how can I fix it please

**Comment by elbakrifz** [comment]
with java 1.8
